test/test-related-lib/TestOverrides.ts

```diff
@@ -0,0 +1,22 @@
+import { EnvOverridable } from "../../src/Constants";
+import os = require('os');
+import path = require('path');
+import { Controller, Services } from "../../src/ioc.config";
+
+export class TestOverrides implements EnvOverridable {
```

Comment:
I've moved out relevant portions to test directory. This class wouldn't be accessible to src directory without changes to `tsconfig.js`.